pkg/storage/mongo/getter.go

```diff
@@ -4,27 +4,27 @@ import (
 	"context"
 	"io"
 
-	"github.com/globalsign/mgo"
-	"github.com/globalsign/mgo/bson"
 	"github.com/gomods/athens/pkg/errors"
 	"github.com/gomods/athens/pkg/observ"
 	"github.com/gomods/athens/pkg/storage"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo/gridfs"
+	"go.mongodb.org/mongo-driver/mongo/options"
 )
 
 // Info implements storage.Getter
 func (s *ModuleStore) Info(ctx context.Context, module, vsn string) ([]byte, error) {
 	const op errors.Op = "mongo.Info"
 	ctx, span := observ.StartSpan(ctx, op.String())
 	defer span.End()
-	c := s.s.DB(s.d).C(s.c)
+	c := s.client.Database(s.db).Collection(s.coll)
 	result := &storage.Module{}
-	err := c.Find(bson.M{"module": module, "version": vsn}).One(result)
+	tctx, cancel := context.WithTimeout(ctx, s.timeout)
+	defer cancel()
+
+	err := c.FindOne(tctx, bson.M{"module": module, "version": vsn}).Decode(result)
```

Comment:
@arpitgogia 
Can you split the calls in two and check for error in each?
Also, please check for `ErrNoDocuments` and if present, return error with `errors.KindUnexpected`
This should also fix the CI error.